polkadot/runtime/parachains/src/scheduler.rs

```diff
@@ -203,6 +204,12 @@ impl<T: Config> Pallet<T> {
 		ValidatorGroups::<T>::decode_len().unwrap_or(0)
 	}
 
+	/// Expected claim queue len. Can be different than the real length if for example we don't have
+	/// assignments for a core.
```

Comment:
`claim_queue_len` is a bit confusing as usually we mean by length of the claim queue the length of individual queues.

If we want to name it with regrads to claim queue, maybe `claim_queue_cores`?